app/resources/en/translation/deceased/dobknown.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Deceased date of birth",
+  "question": "Do you know {deceasedName}'s date of birth?",
```

Comment:
Replace the single quote with `&rsquo;`

Reply:
changed

Reply:
to &apos;

Reply:
Although technically it's an apostrophe, you're right, in Probate we've always used `&rsquo;` rather than `&apos;` ...

Reply:
changed to right ’ to keep in sync with probate